db/migrate/20160927071039_create_notification_blueprints.rb

```diff
@@ -0,0 +1,12 @@
+class CreateNotificationBlueprints < ActiveRecord::Migration
+  def change
+    create_table :notification_blueprints do |t|
+      t.string :group, index: true
+      t.string :level
```

Comment:
What's name used for in http://www.patternfly.org/pattern-library/communication/notification-drawer/#/api?